ops.go

```diff
@@ -73,6 +73,10 @@ func Mv(r *Root, src, dst string) error {
 		return err
 	}
 
+	if srcDirObj == dstDir && srcFname == filename {
```

Comment:
I don't think we should be comparing objects like that, looking at paths should be enough:
```suggestion
	if srcDir == dstDirStr && srcFname == filename {
```